src/settings.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+module.exports = function () {
+    const appSupportPath = path.join(
+        os.homedir(),
+        'Library',
+        'Application Support'
+    )
+    const filePath = path.join(
+        appSupportPath,
+        'com.focusbear.latenomore',
+        'events.json'
+    )
+
+    const data = fs.readFileSync(filePath, 'utf8')
+    return JSON.parse(data)
```

Comment:
can you add default settings (do a JSON merge where the settings from the file override the default settings)? People on older versions won't have everything in their settings file and it will break the app.

Reply:
Done in [e129717](https://github.com/Focus-Bear/late-no-more/pull/19/commits/e129717a62ef29620734b399a3c7756429839333)